lib/src/ui/common/error/error_message_provider.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:allthenews/generated/l10n.dart';
+import 'package:allthenews/src/data/communication/api/api_exception.dart';
+import 'package:flutter/cupertino.dart';
+
+class ErrorMessageProvider {
+  static String provideErrorMessage(Exception exception, BuildContext context) =>
+      provideErrorKey(exception, context);
```

Comment:
jakaś zaszłość ta nazwa? bo w sumie to zwraca odpowiedni string, a nie żaden klucz

Reply:
niefortunna nazwa